.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDTAGS: "include_oss include_gcs"
+      CGO_ENABLED: 1
+      GO111MODULE: "auto"
+      GOPATH: ${{ github.workspace }}
+      GOOS: linux
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: src/github.com/docker/distribution
+        fetch-depth: 50
```

Comment:
@chrispat I think this line is the issue causing DCO script to fail

Reply:
Not it was because git-validation has special handling for Travis https://github.com/vbatts/git-validation/blob/60cb8713913d4fa6c2ca20844a89376c37bd3d35/main.go#L69.  I fixed it by passing in the commit range explicitly.